app/k8s-model.js

```diff
@@ -0,0 +1,22 @@
+const { Model, ModelSet } = require('./base-model');
+const { api } = require('./k8s-api-client');
+
+
+class K8sModel extends Model {
```

Comment:
Is this model actually re-using anything from the original `Model`?

e.g. if for whatever reason we call `K8sModel.create()` it would use the other api, maybe `Model` and `K8sModel` don't have much in common? (I may be wrong/miss something!)

Reply:
Oh, all these `this.prototype.constructor` are referring to the parent class here.

Reply:
It uses the constructor

Reply:
I didn't bother overriding the methods we don't use yet. We might need to override `.create` if we use it in future.

Reply:
`this.prototype.constructor` is equivalent to Python `self.__class__`